app/frontend/src/pages/NotFoundPage/NotFoundPage.tsx

```diff
@@ -0,0 +1,49 @@
+// External Imports
+import React, { Fragment } from "react";
+import { useNavigate } from 'react-router-dom';
+
+// Internal Imports
+import { HeaderNav } from "components/components";
+import { Button } from "components/components";
+import { logoHorizontal, logoStacked, notFoundPageImg } from "assets/images/images";
+
+function NotFoundPage() {
+  const navigate = useNavigate();
+  return (
+    <Fragment>
+      <HeaderNav
+        logoDesktop={logoHorizontal}
+        logoMobile={logoStacked}
+        menu={[
+          { name: "Hack for LA", link: "/" },
+          { name: "How to Join", link: "/qualifier/1" },
+          { name: "Projects", link: "/demo" },
+        ]}
+      />
+      <main>
+        <div className="not-found-container flex-container justify-between align-center gap-3">
+          <div style={{width: "348px"}}>
```

Comment:
Hmm, I'm always conflicted about setting width directly via inline style. Any reasons why this would be preferable? I'm not against it, per se. I am mainly concerned about maintainability in the long run, which is why I discourage inline styling unless it is a last resort.

Reply:
This was just an oversight. Most of my styling experience is with MUI on personal projects where I like to use inline styles to speed up development

Agree with keeping style implementations consistent with the rest of the project